docs/src/markdown/DrawerRailItem.mdx

```diff
@@ -0,0 +1,80 @@
+import Box from '@mui/material/Box';
+
+import RailAnatomy from './images/railAnatomy.png';
+import SharedProps from './SharedProps.mdx';
+
+# Drawer Rail Item
+
+When using the `rail` variant of the `<Drawer>`, you should use `<DrawerRailItem>` instead of `<DrawerNavItem>`. The `<DrawerRailItem>` is a simplified version of the `<DrawerNavItem>` that renders the `icon` and `title` only. When using the `condensed` version of the `<Drawer>`, the `title` will also be hidden. The `<DrawerRailItem>` cannot be nested.
+
+<Box
+    sx={{
+        width: '100%',
+        textAlign: 'center',
+    }}
+>
+    <Box
+        sx={{
+            width: '100%',
+        }}
+        component="img"
+        alt="Rail Anatomy"
+        src={RailAnatomy}
+    />
+</Box>
+
+### Drawer Rail Item API
+
+<Box>
+
+| Prop Name                       | Description                                                                                     | Type                    | Required | Default |
+| ------------------------------- | ----------------------------------------------------------------------------------------------- | ----------------------- | -------- | ------- |
+| classes                         | Style overrides                                                                                 | `DrawerRailItemClasses` | no       |         |
+| condensed\*                     | Show condensed nav items without labels (`rail` variant only)                                   | `boolean`               | no       | `false` |
+| hidden                          | Sets whether to hide the rail item                                                              | `boolean`               | no       |         |
+| icon                            | A component to render for the left icon                                                         | `JSX.Element`           | yes      |         |
+| itemID                          | An unique identifier of the NavItem. Item will have 'active' style when this matches activeItem | `string`                | yes      |         |
+| onClick                         | A function to execute when clicked                                                              | `function`              | no       |         |
+| statusColor                     | Status stripe and icon color                                                                    | `string`                | no       |         |
+| title                           | The text to show on the first line                                                              | `string`                | no       |         |
+| ButtonBaseProps                 | Used to override [ButtonBase](https://material-ui.com/api/button-base/) default props           | `InfoListItemProps`     | no       |         |
```

Comment:
May want to point this link to the latest MUI documentation

Reply:
Done.